src/main/java/blackjack/domain/card/Hand.java

```diff
@@ -8,24 +8,43 @@
 
 public class Hand {
 
+    private static final Score BLACKJACK_SCORE = new Score(21);
+    private static final int BLACKJACK_CARD_COUNT = 2;
+
     private final List<Card> hand = new ArrayList<>();
 
     public void add(Card card) {
         hand.add(card);
     }
 
-    public Score sum() {
+    public boolean isBlackJack() {
+        return calculateScore().equals(BLACKJACK_SCORE)
```

Comment:
```suggestion
        return BLACKJACK_SCORE.equals(calculateScore())
```
객체 동등성 비교를 할 때는 `null이 될 수 없는 것`.equals(~~) 방식이 더 좋다고 생각합니다~!

Reply:
앗 생각을 못했던 부분입니다!!
세심한 피드백 감사합니다!!
수정하였습니다!!